spark/src/main/scala/org/apache/spark/sql/delta/sources/DeltaSQLConf.scala

```diff
@@ -423,6 +423,14 @@ trait DeltaSQLConfBase {
       .checkValues(Set(1, 2, 3))
       .createWithDefault(1)
 
+  val TABLE_FEATURES_TEST_FEATURES_ENABLED =
+    buildConf("tableFeatures.testFeatures.enabled")
+      .internal()
+      .doc("Controls whether test features are enabled in testing mode. " +
+        "This config is only used for testing purposes. ")
+      .booleanConf
+      .createWithDefault(true)
+
```

Comment:
Do we still need this? I see we use this in TableFeature, but we never use it in tests AFAICT.

Reply:
Should we double guard this so by default it's also only enabled in testing mode?

Reply:
It turns out I need to wrap this with an ugly try block because there are cases where the `conf` is not initialized...